registry.json

```diff
@@ -436,6 +471,11 @@
     "listed": true,
     "version": "4.3.0"
   },
+  "StrongInject": {
+    "listed": true,
+    "version": "1.0.0",
```

Comment:
Please don't use version 1.0.0. Not only is this missing a lot of features and minor bug fixes, it was also deprecated from nuget due to a significant bug.

I would suggest using the latest major version as of this point in time: 1.3.0

Reply:
The version listed here is the minimum version supported in Unity. This will let people install every version starting from 1.0.0

Reply:
Got it thanks